tests/testthat/test-calculateNotApplicableStatus.R

```diff
@@ -4,19 +4,23 @@ test_that("Not Applicable status Table Empty", {
   outputFolder <- tempfile("dqd_")
   on.exit(unlink(outputFolder, recursive = TRUE))
 
+  # Make sure the device exposure table is empty
+  connection <- DatabaseConnector::connect(connectionDetailsEunomia)
```

Comment:
should this be connectionDetailsEunomiaNaChecks?

Reply:
Yes, thanks, fixed now.